.github/workflows/test.yml

```diff
@@ -32,40 +34,42 @@ jobs:
         uses: actions/checkout@v3
       - name: Install apt packages
         run: "apt-get update && apt-get upgrade -y && apt-get -y install wget zip unzip sudo && apt-get clean && echo 'Installed linux dependencies'"
-      - name: Install node 13
-        run: curl -sL https://deb.nodesource.com/setup_12.x | bash - && apt-get update && apt-get install -y nodejs
+      - name: Install node 16
+        run: curl -sL https://deb.nodesource.com/setup_16.x | bash - && apt-get update && apt-get install -y nodejs
       - name: Install dependencies and link
         run: npm ci && npm link
       - name: Run tests
         run: npm run test
 
-  test_java14_latest_node_16:
-    name: Test - Debian - AdoptOpenJDK latest - Node 16
+  test_java_latest_node_16:
+    name: Test - Debian - OpenJDK latest - Node 16
     runs-on: ubuntu-latest
     container:
-      image: adoptopenjdk:latest
+      image: eclipse-temurin:latest
```

Comment:
The [adoptopenjdk Docker images](https://hub.docker.com/_/adoptopenjdk) are now deprecated in favor of the eclipse-temurin image.

I was not sure if you wanted this job to run Java 14 or the latest version of Java available. `eclipse-temurin:latest`currently includes Java 20.